drivers/bluetooth/hci/Kconfig

```diff
@@ -109,6 +109,19 @@ config BT_STM32_IPM_RX_STACK_SIZE
 	depends on BT_STM32_IPM
 	default 512
 
+if BT_H4
+
+config BT_CYW43XXX
+	bool "CYW43XXX BT connectivity"
+	select GPIO
```

Comment:
Select is discouraged for entire subsystems, there's been few discussions about it recently. Not a big deal for `GPIO` specifically but this should be a `depends on GPIO` instead.

Also since you have a matching device tree entry you should use

```
default y
depends on DT_INFINEON_CYW43XXX_BT_HCI_ENABLED
```

So that you don't have to select it manually in the project configs.

Reply:
updated. Thanks!